### ISSUE TITLE
ability to set map options

### ISSUE THREAD

**Issue by bcbirkhauser** [Vheissu/aurelia-google-maps#34]
Is there a way for me to set the map options? I'm particularly interested in the ability to set the styles of the map. https://developers.google.com/maps/documentation/javascript/styling

**Comment by bartlenaerts** [comment]
Yes. I've integrated this in version 1.0.15. Use `options` as an extra property in the `config.options` method.

``` Javascript
aurelia.use.plugin('aurelia-google-maps', config => {
  config.options({
    apiKey: 'myapiKey',
    apiLibraries: 'drawing,geometry',
    options: { styles: [/*your styles here*/] }
  });
});
```

**Comment by justindra** [comment]
Can we set this so we can add it when setting each instance? i.e.

`<google-map options="someOptionsObject"></google-map>`

Nice to be able to remove certain functionality like scrolling, panning etc. on some instances, but not all.

**Comment by bartlenaerts** [comment]
Just added the possibility to add options to each custom element separate, to https://github.com/aurelia-plugins/aurelia-plugins-google-maps.

**Comment by justindra** [comment]
Just added a PR, https://github.com/Vheissu/aurelia-google-maps/pull/39

**Comment by Vheissu** [comment]
This has just been released in 1.0.16 guys. Thanks for your contributions.
